src/browser/main.js

```diff
@@ -1369,6 +1372,7 @@
             "hda": settings.hda,
             "hdb": settings.hdb,
             "cdrom": settings.cdrom,
+            "wants_cdrom": settings.wants_cdrom,
```

Comment:
I think `cdrom: { ejected: true }` would be a clearer api for this.

Reply:
I’m happy to defer to you here, I’ll make that change.

Reply:
I think `cdrom: { ejected: true }` makes sense for the starter options, but it's a bit tricky inside of cpu and so on if the `settings` object is either a buffer or null or `{ejected: true}`.  Should there be a new `EmptyBuffer` buffer type?  Or can `wants_cdrom` continue to exist within the emulator settings, even if it's not part of public API?

Reply:
Yes, internally `wants_cdrom` is fine (it can be refactored at any point anyway)